components/Login/Login.tsx

```diff
@@ -106,8 +110,7 @@ function Login({ navigation }: LoginProps) {
   const authenticate = async () => {
     if (validateEmail() && validatePassword()) {
       try {
-        const user = await Auth.signIn(email, password);
-        console.log(user);
+        await Auth.signIn(email, password);
```

Comment:
I assume we can get the currently logged in user from the api so no need to save it here?

Reply:
Yeah, that's right